test/ClangImporter/Inputs/custom-modules/ObjCIRExtras.h

```diff
@@ -60,8 +60,12 @@
 @interface SwiftNameTestErrorSub : SwiftNameTestError
 @end
 
+@interface SwiftGenericNameTest<T> : NSObject
```

Comment:
Please include a test where T is constrained (either to a class or to a protocol or both).

Reply:
I think particular case T is constrained to NSObject, I have a test which tried to pass `Int` to it and fails, but I'll add more too.

Reply:
I've added one more class which has T constrained to NSNumber *

Reply:
I believe it's just constrained to AnyObject. NSNumber is good enough!